app/views/qsets/show_questions.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="container-fluid" id="qset-show-container" data-qset-id="<%= @qset.id %>">
+  <%= render "qset_context" %>
+  <div id="generate-question-button-filter">
+    <%= render "generate_question" %>
+  </div>
+  <div id="all-mine-filter-container">
+    <%= render "all_mine_other_filter" %>
+  </div>
+  <div class="row">
+    <div class="col-sm-12">
+      <div id ="generate-question-button-filter-xs">
+        <%= render "generate_question" %>
+      </div>
+      <div id="all-mine-filter-container-xs">
+        <%= render "all_mine_other_filter" %>
+      </div>
+      <%= render "question_list" %>
+    </div>
+  </div>
+</div>
+
+<%= render "modal_question" %>
+<%= render "modal_social" %>
```

Comment:
We're no longer including a link to the social modal, so I think we don't need to render this.